src/Commands/InitCommand.php

```diff
@@ -349,12 +363,15 @@ protected function addQuotes($string): string
         return (Str::contains($string, ' ')) ? "\"{$string}\"" : $string;
     }
 
-    protected function publishMigration(): void
+    protected function publishMigration(View $view, string $migrationName): void
     {
-        $data = view('initializator::add_default_user')->with($this->adminCredentials)->render();
-        $fileName = Carbon::now()->format('Y_m_d_His') . '_add_default_user.php';
+        $time = Carbon::now()->format('Y_m_d_His');
+
+        $migrationName = "{$time}_{$migrationName}.php";
+
+        $data = $view->render();
 
-        file_put_contents("database/migrations/{$fileName}", "<?php\n\n{$data}");
+        file_put_contents("database/migrations/{$migrationName}", "<?php\n\n{$data}");
```

Comment:
Let's add last empty line as required by PSR-12 https://www.php-fig.org/psr/psr-12/#22-files

All PHP files MUST end with a non-blank line, terminated with a single LF.

```suggestion
        file_put_contents("database/migrations/{$migrationName}", "<?php\n\n{$data}\n");
```